build/configs/esp_wrover_kit/STDK/defconfig

```diff
@@ -1145,6 +1145,7 @@ CONFIG_STDK_MQTT_USE_SSL=y
 CONFIG_STDK_IOT_CORE_USE_MBEDTLS=y
 # CONFIG_STDK_IOT_CORE_CRYPTO_SUPPORT_RSA is not set
 CONFIG_STDK_IOT_CORE_CRYPTO_SUPPORT_ED25519=y
+CONFIG_MBEDTLS_USER_FILE="/root/tizenrt/external/stdk/tizenrt-wrapper/mbedtls_config.h"
 # CONFIG_STDK_IOT_CORE_FS_SW_ENCRYPTION is not set
```

Comment:
how this subpath "/root/tizenrt" works?

Reply:
it's the absolute path in docker env.
as we use this Kconfig string type, which always have "", so we can't use a global path with <>, and could only use such kind of absolute full path.